case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/case-study-azure-devops.html
+folder: development-platforms
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Pricing](#pricing)
+7. [Comparison](#comparison)
+8. [How To](#how-to)
+9. [Summary](#summary)
+10. [Credits](#credits)
+11. [References](#references)
+
+## **Introduction**
+
+<div style="width: 600px;">
+
+![Azure DevOps offerings](https://miro.medium.com/max/800/1*8orwInnxqPRhrcKf9aOo9Q.png)
+
+<p style="padding: 0; margin: 0;">
+
+*Source: google*
+
+</p>
+
+</div>
+
+**Azure DevOps** was initially released in 2018 by Microsoft. It is a set of developer tools, services, and features that enable teams to plan, develop, test, and deliver software more effectively. Azure DevOps integrates with the broader Azure cloud computing platform, enabling users to take advantage of its features and services, such as continuous integration and deployment, and virtual machines. Azure DevOps has since become one of the leading development platforms and is widely used by development teams across the world.
+
+Azure DevOps is for the most part free to use, however there is some limitations with using only a free tier option. I will go in deeper details about the pricing [here](#pricing)
+
+## **Brief History**
+
+In 2005 the service that we today refer to as Azure DevOps was named Team Foundation server(TFS). It was released as a set of development tools and services for teams building software applications. After its release in 2005 it has evolved and expanded with new features and services and have quickly become the most popular development platforms for Microsoft-centric teams. TFS has been around for more than a decade, and it has evolved a lot since its inception in 2005. There are professionals in the industry whose entire careers have been dedicated to managing TFS. Such expertise has been necessary because it was quite complicated to administer.
+
+In 2012, TFS morphed into a tool that helped teams manage their software development projects using Agile. [[1](#references)]
+
+The name **"Azure DevOps"** came in 2018. Microsoft re-branded TFS to Azure DevOps, this reflecting Microsoft shift to a cloud based service and its focus on providing organizations with a more comprehensive set of tools and services.
+After the re-branding Azure DevOps has continued to evolve and add more features, you can read more about the features [here](#features). Azure DevOps has also integrated other Azure services such as Azure functions, Azure Container and Azure Kubernetes service. [[2](#references)]
+
+## **Features**
```

Comment:
```suggestion
## Features
```

The CSS file for the renderer uses heavy bold for headings so this is unnecessary and can be removed.